api/python/quilt3/bucket.py

```diff
@@ -159,8 +159,8 @@ def ls(self, path=None, recursive=False):
             recursive (bool): show subdirectories and their contents as well
 
         Returns:
-            ``list``: Return value structure has not yet been permanently decided
-            Currently, it's a ``tuple`` of ``list`` objects, containing the
+            `list`: Return value structure has not yet been permanently decided
```

Comment:
This change seems unrelated to testing codeblocks. I'd prefer unrelated changes to go to separate PRs.

Reply:
The triple-quotes make the parser think it is a codeblock.